### PR TITLE
Remove listenbuffer

### PR DESCRIPTION
**Pull request by LK4D4**
SOMAXCONN should be configured by machine administrator.

### CODE REVIEW

**Comment by calavera** [comment]
Agreed. LGTM

File: pkg/sockets/unix_socket.go [hunk 1]
```diff
@@ -10,20 +10,19 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/listenbuffer"
 	"github.com/opencontainers/runc/libcontainer/user"
 )
 
 // NewUnixSocket creates a unix socket with the specified path and group.
 // The channel passed is used to activate the listenbuffer when the caller is ready
```
Comment (estesp) [inline pkg/sockets/unix_socket.go]: comment should be updated to remove mention of listenbuffer-based functionality

**Comment by LK4D4** [comment]
@estesp Thanks! Fixed.

**Comment by jfrazelle** [comment]
LGTM

**Comment by estesp** [comment]
LGTM

**Pull request merged**